osu.Game/Screens/Multi/Lounge/Components/DrawableRoom.cs

```diff
@@ -232,5 +236,10 @@ private void load()
                 Current = name;
             }
         }
+
+        public MenuItem[] ContextMenuItems => new MenuItem[]
+        {
+            new OsuMenuItem("Duplicate", MenuItemType.Standard, () => DuplicateRoom?.Invoke(Room))
```

Comment:
not sure with naming this option *Duplicate* - would prefer something like *Create copy* myself